test/unit/math/test_ad.hpp

```diff
@@ -1435,14 +1435,36 @@ void expect_ad_vectorized_binary(const F& f, const T1& x, const T2& y) {
 }
 
 /**
- * Runs `expect_near_rel` over the values and adjoints of `var_value` types.
+ * Runs `expect_near_rel` over the values and adjoints of
+ *  `std::vector<var_value<T>>` types.
  * @tparam T1 A `var_value` with any template parameter
  * @tparam T2 A `var_value` with any template parameter
  * @param x A `var_value` whose values and adjoints are compared against `y`s
  * @param y A `var_value` whose values and adjoints are compared against `x`s
  */
 template <typename T1, typename T2,
-          require_all_var_t<scalar_type_t<T1>, scalar_type_t<T2>>* = nullptr>
+          require_all_std_vector_st<is_var, T1, T2>* = nullptr>
+void expect_near_rel_var(const std::string& message, T1&& x, T2&& y,
+                         const ad_tolerances& tols) {
+  stan::math::check_size_match("expect_near_rel_var", "x", x.size(), "y",
+                               y.size());
+  for (size_t i = 0; i < x.size(); ++i) {
+    expect_near_rel(message + std::string(" values"), x[i].val(), y[i].val(),
+                    tols.gradient_val_);
```

Comment:
[optional] might be nice to have something like 

```suggestion
    expect_near_rel(message + std::string(" values"), x[i].val(), y[i].val(),
                    tols.gradient_val_) << "Failed at (i): " << i;
```